spec/14-classes.md

```diff
@@ -387,6 +387,14 @@ Widget::__callStatic('sMethod', array(NULL, 1.234))
 <pre>
   <i>const-declaration:</i>
     const  <i>name</i>  =  <i>constant-expression</i>   ;
+
+  <i>class-const-declaration:</i>
```

Comment:
Just noticed that these productions only allow a single constant, while it's actually possible to define multiple (but this is an existing issue, you don't need to fix it).